context/AuthHelper.tsx

```diff
@@ -59,46 +44,31 @@ export const loginAccount = async ({
 };
 
 /**
- * Get user data from the session
- * @param router - router to navigate the page
- * @param updateUser - updateUser() function
- * @param resetUser - resetUser() function
- * @param setIsSignedIn - changes to false if errors
+ * Log out and clear session state
  * @returns {Promise<void>}
  */
-export const getUser = async (
```

Comment:
Why are we removing getUser?

Reply:
I think this was a part of the code that was put in that wasn't in alex's branch